plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -98,7 +98,9 @@ class ContainerServiceFormSection extends Component {
     const inputFiled = (
       <FieldInput
         key="gpus-input"
+        min="0.001"
```

Comment:
Let's change this to `0` as it is specified as an int32... 

https://github.com/mesosphere/marathon/blob/master/docs/docs/rest-api/public/api/v2/types/resources.raml